README.md

```diff
@@ -27,21 +29,20 @@ Washington, DC 20001, United States
 
 ## Schedule
 
-|  Time (EST) | Topic                                                   |
-|:-----------:|:-------------------------------------------------------:|
-| 08:30-08:40 | Introduction to GMT / PyGMT                             |
-| 08:40-08:55 | Installation                                            |
-| 08:55-09:45 | Tutorial 1 - First figure + Subplots / layout           |
-| 09:45-10:15 | **Break**                                               |
+| Time (EST)  |                                     Topic                                     |
+| :---------: | :---------------------------------------------------------------------------: |
+| 08:30-08:40 |                          Introduction to GMT / PyGMT                          |
+| 08:40-08:55 |                                 Installation                                  |
+| 08:55-09:45 |                 Tutorial 1 - First figure + Subplots / layout                 |
```

Comment:
To be consistent with the titles of the tutorials (sorry for destroying the linting...).

```suggestion
| 08:55-09:45 |                Tutorial 1 - First figure and Subplots / layout               |
```